Detections/DSMAzureBlobStorageLogs/AccessToSensitiveDataFromKnownTiMatchedIpAdresses.yaml

```diff
@@ -0,0 +1,86 @@
+id: e3c63c74-1140-406b-bfd6-9ddee2677f9e
+name: Access to sensitive data from known TI matched IP addresses
+description: |
+  This query detects access to data that has classification insights (link below) and/or sensitivity label insights (link below), from IP adresses that are marked via the TI.
+  Classification insights: https://docs.microsoft.com/en-us/azure/purview/classification-insights
+  Sensitivty label insigths: https://docs.microsoft.com/en-us/azure/purview/sensitivity-insights
+requiredDataConnectors: []
```

Comment:
Seems like we might need the ThreatIntelligence connectorid details here

Reply:
It looks like I switched it, and we need it only here, instead of the other one... doesn't it?

Reply:
@amramz - You are good here.